### PR TITLE
Feature/bi 12473 image rendering functionality

### PR DESCRIPTION
**Pull request by adeharrison-ch**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -233,6 +233,9 @@
                         <exclude>**/uk.gov.companieshouse.account.validator.Runner.java</exclude>
                     </excludes>
                     <skipTests>${skip.unit.tests}</skipTests>
+                    <environmentVariables>
+                        <IXBRL_TO_PDF_URI>http://random.api.url</IXBRL_TO_PDF_URI>
```
Comment (modhasanjay) [inline pom.xml:237]: Is this going to be changed to the correct url?

File: src/main/java/uk/gov/companieshouse/account/validator/controller/AccountValidationController.java [hunk 2]
```diff
@@ -95,6 +111,40 @@ ResponseEntity<?> getStatus(@PathVariable final String fileId) {
...
+
+        String iXbrlToPdfUri = getIxbrlToPDFEnvVal();
+        if (StringUtils.isBlank(iXbrlToPdfUri)) {
+            throw new MissingEnvironmentVariableException(String.format("Missing '%s' environment variable", IXBRL_TO_PDF_URI_KEY));
+        }
+
+        ByteArrayResource contentsAsResource = new ByteArrayResource(file.get().getData()) {
+            @Override
+            public String getFilename() {
+                return "anything";
```
Comment (modhasanjay) [inline src/main/java/uk/gov/companieshouse/account/validator/controller/AccountValidationController.java:135]: Is this "anything" going to be changed to a correct string?

**Review by modhasanjay — COMMENTED** [review]
Please check review comments.

File: pom.xml (hunk 1 above)
Comment (adeharrison-ch) [reply pom.xml:237]: no, it's only for testing purposes

File: src/main/java/uk/gov/companieshouse/account/validator/controller/AccountValidationController.java (hunk 2 above)
Comment (adeharrison-ch) [reply src/main/java/uk/gov/companieshouse/account/validator/controller/AccountValidationController.java:135]: no, doesn't make any difference, required when posting byte[] as per https://medium.com/@voziv/posting-a-byte-array-instead-of-a-file-using-spring-s-resttemplate-56268b45140b - stumped me for quite a while

**Review by modhasanjay — APPROVED** [review]
ok

**Pull request merged**